libcontainer/configs/namespaces_unix.go

```diff
@@ -96,13 +96,13 @@ func (n *Namespaces) Remove(t NamespaceType) bool {
 	return true
 }
 
-func (n *Namespaces) Add(t NamespaceType, path string) {
+func (n *Namespaces) Add(t NamespaceType, path string) bool {
 	i := n.index(t)
 	if i == -1 {
 		*n = append(*n, Namespace{Type: t, Path: path})
-		return
+		return true
 	}
-	(*n)[i].Path = path
+	return false
 }
```

Comment:
At the beginning, I just mistakenly thought the last line of code in `Add()` is redundant. But now it seems that my PR is meaningless and it's better to remain unchanged. Thanks for all your comments. I will close this PR later.